iosApp/iosApp.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -159,7 +159,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/davidstump/SwiftPhoenixClient",
       "state" : {
-        "revision" : "4e548e4ce4d46574a20b8a539d52d367fb0eae24"
+        "revision" : "25a4d8a3fa754ef3fb587a34addd08c7e568abc4",
+        "version" : "5.3.5"
```

Comment:
This was pinned a[ branch of  SwiftPhoenixClient with my changes](https://github.com/davidstump/SwiftPhoenixClient/pull/260) to address some socket race conditions on backgrounding. I didn't follow-through on that PR, but the core of the changes were added in 5.3.4 https://github.com/davidstump/SwiftPhoenixClient/commit/a21c46578aab377a559ce6c29b387bd5a451be09